src/Components/NavBar.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import '../NavBar.css'
+import { NavLink } from 'react-router-dom';
```

Comment:
Decide between `' '` or `" "` and whether to end statements with a semi-colon or not. :)